textbook/03/5/2/Functions.ipynb

```diff
@@ -6,9 +6,311 @@
     "tags": []
```

Comment:
<p>If we want to more explicit -&gt; If we want be to more explicit</p><p>about which variable in which in  -&gt; about which variable is which in</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>